fpga_interchange/add_prim_lib.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+""" Utility for adding a primitive library to a device from Yosys JSON """
```

Comment:
Please add a small bit of instructions on how to generate the Yosys JSON for just the library cells for the relevant arch.

Reply:
Done